hearth/sig/lib/hearth/plugin_list.rbs

```diff
@@ -0,0 +1,14 @@
+module Hearth
```

Comment:
You can use rbs prototype command to generate these. Not sure if that's what you did here. We should probably have a discussion if any of the rbs stuff is worth it given the adoption has been pretty low.

Reply:
Yeah, the prototype command didn't seem to give me very good types, so I  scarped it and hand wrote it.  

But yeah, worth revisiting our rbs strategy....

Reply:
For now, can you run the prototype command and just use it as is?

Reply:
Sure - its less useful since its all just untyped, but given lack of general support/usage of rbs, thats probably fine.

Reply:
Yeah. I think just generally it'd be nice to figure out what to do with them and then make a sweeping change .. for now I think they're just skeletons.. I think IDE actually picks up the types without it anyway?